### ISSUE TITLE
Need more clear errors instead of "format for LanguageID not supported"

### ISSUE THREAD

**Issue by vanarok** [mattn/efm-langserver#273]
If there are errors in the formatter or linter, efm will generate a misleading error: "format for LanguageID not supported: any-lang".

**Comment by mattn** [comment]
What is better?

**Comment by mattn** [comment]
Ah, sorry. Thanks for let me known. Can you make pull-request to fix issue?

**Comment by mattn** [comment]
Agreed the message was confusing. On current releases the formatter path only returns this error when no formatter config matched; command failures are logged (run with `-loglevel 3 -logfile ...` to see the command output). Closing; please open a fresh issue if a concrete case still produces a misleading error on a current release.

**Comment by mattn** [comment]
Sorry for the very late response. Closing this as it is old and can no longer be reproduced. If you still hit the same problem on a current release, please open a new issue.
